keras_cv/layers/preprocessing/random_crop_and_resize_test.py

```diff
@@ -209,16 +215,15 @@ def test_augment_bbox(self):
             output = layer(input, training=True)
 
         expected_output = np.asarray(
-            [[[0, 0, 10, 10, 0], [0, 0, 6, 6, 0], [6, 6, 8, 8, 0]]]
+            [[0, 0, 10, 10, 0], [0, 0, 6, 6, 0], [6, 6, 8, 8, 0]]
         )
-        expected_output = np.reshape(expected_output, (1, 3, 5))
         self.assertAllClose(expected_output, output["bounding_boxes"])
 
     def test_augment_bbox_batched_input(self):
         image = tf.zeros([20, 20, 3])
 
         bboxes = tf.convert_to_tensor(
-            [[[0, 0, 10, 10], [4, 4, 12, 12]], [[4, 4, 12, 12], [0, 0, 10, 10]]]
+            [[0, 0, 10, 10], [4, 4, 12, 12]], [[4, 4, 12, 12], [0, 0, 10, 10]]
```

Comment:
This format will not be converted

Reply:
You could use something like `bboxes= tf.constant([[[0, 0, 10, 10], [4, 4, 12, 12]], [[4, 4, 12, 12], [0, 0, 10, 10]]])`

Reply:
Good idea, will update in the next batch of changes!